go.mod

```diff
@@ -2,12 +2,15 @@ module github.com/dlespiau/footloose
 
 require (
 	github.com/ghodss/yaml v1.0.0
-	github.com/inconshreveable/mousetrap v1.0.0 // indirect
+	github.com/kr/pretty v0.1.0 // indirect
```

Comment:
It looks like you may have imported a dependency at some point. This happens to me with my IDE trying to be clever (with goimport) and start importing things form my GOTPATH. And then building the package modifies `go.mod` and `go.sum`.

This should be cleaned up, I don't believe we need any change in `footloose` dependencies for this change.